NPE with HBase admin create table with column descriptor versions Integer.MAX_VALUE

The NPE surfaced in junit test I'm working on with the bigtable emulator. The test was running from a build from master. 

In HBase, when creating a table with unlimited versions, users would set the max versions to Integer.MAX_VALUE. This isn't widely used for production cluster as some limit is usually set, but NPE wouldn't be thrown. When setting versions to Integer.MAX_VALUE - 1, there is no issue. 

Impact: low 
Workaround: Integer.MAX_VALUE - 1

#### Code example

```java
    Configuration bigtableConf = new Configuration(false);
    BigtableConfiguration.configure(bigtableConf, BT_TEST_PROJECT, BT_TEST_INSTANCE);
    bigtableConf.set(
        BigtableOptionsFactory.BIGTABLE_EMULATOR_HOST_KEY,
        "localhost:" + bigtableEmulator.getPort());
    Connection bigtableConn = BigtableConfiguration.connect(bigtableConf);

    Admin admin = bigtableConn.getAdmin();
    HTableDescriptor htd = new HTableDescriptor(BT_TEST_TABLE);
    HColumnDescriptor columnDesc = new HColumnDescriptor(TEST_CF);
    columnDesc.setMaxVersions(Integer.MAX_VALUE);
    htd.addFamily(columnDesc);
    admin.createTable(htd);
```

#### Stack trace
```
[ERROR] runImportTest(com.google.cloud.bigtable.mapreduce.hbasesnapshots.TestEndToEndIT)  Time elapsed: 2.069 s  <<< ERROR!
java.lang.NullPointerException
	at com.google.bigtable.repackaged.com.google.cloud.bigtable.admin.v2.models.CreateTableRequest.addFamily(CreateTableRequest.java:68)
	at com.google.cloud.bigtable.hbase.adapters.admin.TableAdapter.adapt(TableAdapter.java:49)
	at com.google.cloud.bigtable.hbase.adapters.admin.TableAdapter.adapt(TableAdapter.java:56)
	at org.apache.hadoop.hbase.client.AbstractBigtableAdmin.createTable(AbstractBigtableAdmin.java:334)
	at org.apache.hadoop.hbase.client.AbstractBigtableAdmin.createTable(AbstractBigtableAdmin.java:303)
	at com.google.cloud.bigtable.mapreduce.hbasesnapshots.TestEndToEndIT.bigtableSetup(TestEndToEndIT.java:128)
	at com.google.cloud.bigtable.mapreduce.hbasesnapshots.TestEndToEndIT.setup(TestEndToEndIT.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
        ...
```

--- a/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/admin/TableAdapter.java
+++ b/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/admin/TableAdapter.java
@@ -20,6 +20,7 @@ import static com.google.cloud.bigtable.hbase.adapters.admin.ColumnDescriptorAda
 import com.google.api.core.InternalApi;
 import com.google.cloud.bigtable.admin.v2.models.ColumnFamily;
 import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.admin.v2.models.GCRules.GCRule;
 import com.google.cloud.bigtable.admin.v2.models.Table;
 import com.google.protobuf.ByteString;
 import org.apache.hadoop.hbase.HColumnDescriptor;
@@ -46,7 +47,12 @@ public class TableAdapter {
     if (request != null) {
       for (HColumnDescriptor column : desc.getColumnFamilies()) {
         String columnName = column.getNameAsString();
-        request.addFamily(columnName, buildGarbageCollectionRule(column));
+        GCRule gcRule = buildGarbageCollectionRule(column);
+        if (gcRule == null) {
+          request.addFamily(columnName);
+        } else {
+          request.addFamily(columnName, gcRule);
+        }
       }
     }
   }

